c2corg_ui/static/js/documentediting.js

```diff
@@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
     scope: true,
     controller: 'appDocumentEditingController',
     controllerAs: 'editCtrl',
-    bindToController: true
+    bindToController: true,
+    link: function(scope, el, attrs, ctrl) {
+      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
```

Comment:
The problem is not that it will appear 0.5 seconds later. The problem is that you have no guarantee that 500ms are are always long enough. Are the elements that you want to count always ready at that moment?

It would be good if you could avoid this timeout.